tools/benchmark/cmd/mvcc-put.go

```diff
@@ -94,6 +94,7 @@ func mvccPutFunc(_ *cobra.Command, _ []string) {
 		}
 
 		defer func() {
+			f.Close()
```

Comment:
The following `pprof.WriteHeapProfile(f)` will run into issue because the file has already been closed? So suggest to add a `defer f.Close()` above (right after the `if err !=nil` check).